Features/net60

mstancombe opened the pull request:
hey @driekus77 , I was working on a branch of Html-Renderer and had the SkiaSharp stuff mostly working.  However, I was working off a fork of a fork, and then went back to the ArthurHub repo and noticed your PR.

Great work!

I've integrated some of the fixes for Skia that I'd done in my branch into yours, so the PR should fix a heap of things with that.
Still haven't done all of it, but it's a lot closer now :)

mstancombe committed:
Merge pull request #2 from driekus77/features/net60

Introducing net6.0 port of HtmlRenderer

mstancombe committed:
Adding demo app to compare, Modify PdfSharp to use Async invocation, WIP on Skia.

mstancombe committed:
Updated to .net 7, lots more work getting Skia font sizing more accurately matching PDFSharp, Implemented lines/rectangles in Skia.

mstancombe committed:
Updated remaining projects to net7, some tweaks for winforms sample.

driekus77 commented:
Hee Thanks @mstancombe. I must admit not doing al lot anymore with this lib. Do you expect some work from me?
Nice work!

mstancombe commented:
@driekus77 No expectation, just thought you might want these changes in your repo :)

I'm happy to keep plugging away in mine, and if you ever want to pull anything in from it, feel free.

Maybe one day we can merge all this back into the original fork! but yeah, not likely 😅

mstancombe committed:
Update skia dependency

mstancombe committed:
Adding build pipeline for the net6 branch to push Core and Skia.

mstancombe committed:
Add project URL

mstancombe committed:
Set build scripts to net 7

mstancombe committed:
Removed non cross plat rendering targets (Win/WPF) from default builds.  These might just be removed permenantly in future.

mstancombe committed:
Fix case issue

mstancombe committed:
Fix issue where core not published

mstancombe committed:
Image Fixes
-Implemented image loading using HttpClient
-Added demo resources images to console program
-Moved sample test png expected location out of root c: folder.  Ideally this should be something less platform specific, but haven't crossed that bridge yet.

mstancombe committed:
Add SVG Support

mstancombe committed:
Fix issue where images not waiting for load.

mstancombe committed:
Add basic page-break 'always' support, remove default page breaking on printed h1 element.

Commit:
Swap skia svg package

mstancombe committed:
Merge pull request #3 from albyoo/features/net60

Swap Skia SVG NuGet Package

Commit:
Rename corner radius to border radius

Commit:
Update demo output to use date format yyyyMMdd

Commit:
Implement skia border radius

mstancombe committed:
Merge pull request #4 from albyoo/features/border-radius

Rename corner-radius to border-radius & Implement Skia border-radius.

Commit:
Update graphics adapter to use correct calculated height

Commit:
Update skia sharp adapter to use font manager default singleton

mstancombe committed:
Merge pull request #6 from albyoo/features/font-sizing

Update font size caluclation

driekus77 commented:
Didn't have time to test it but wanted to save the work!
